portfolio/src/main/webapp/script.js

```diff
@@ -18,6 +18,7 @@
 
 function getMessage() {
   fetch('/data').then(response => response.text()).then(msg => {
+    console.log(msg);
```

Comment:
Shouldn't you use `response.json()` now? The server is returning a json.

Reply:
not sure, I was just following what they said in the walkthrough

Reply:
The walkthrough mentioned parsing json: https://github.com/connorbrett/step/blob/master/walkthroughs/week-3-server/step-3-json-walkthrough.md#parsing-json

Reply:
ok that makes more sense, misread it the first time i went through it, fixed now